tests/e2e/prometheus-config-validation/03-promreceiver-nopromconfig.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   mode: statefulset
   targetAllocator:
     enabled: true
-    image: "local/opentelemetry-operator-targetallocator:e2e"
+    image: "ghcr.io/iblancasa/opentelemetry-operator/target-allocator:v0.81.0-14-g27e2a7e"
```

Comment:
Shouldn't we be using the local image here?

Reply:
Sorry. I added changes that should not be in the PR. Fixing...